### PR TITLE
Feature/idcat mòbil

### PR DESCRIPTION
**Pull request by tramuntanal**
Add OAuth2 registration/login feature via AOC's VALid: IdCat mòbil.

Related with #15

### CODE REVIEW

File: config/initializers/omniauth_idcat_mobil.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Devise.setup do |config|
+  config.omniauth :idcat_mobil,
+                  ENV["IDCAT_MOBIL_CLIENT_ID"],
+                  ENV["IDCAT_MOBIL_CLIENT_SECRET"],
+                  ENV["IDCAT_MOBIL_SITE_URL"],
+                  scope: :autenticacio_usuari
```
Comment (isaacmg410) [inline config/initializers/omniauth_idcat_mobil.rb]: in english?

Comment (tramuntanal) [reply config/initializers/omniauth_idcat_mobil.rb]: no, IdCat mòbil requires this parameter to be in catalan.

**Comment by tramuntanal** [comment]
@isaacmg410 no, keeping in synch with master is ok as we'll probably publish with 0.13.

File: config/initializers/omniauth_idcat_mobil.rb (hunk 1 above)
Comment (isaacmg410) [reply config/initializers/omniauth_idcat_mobil.rb]: ok

**Comment by isaacmg410** [comment]
@tramuntanal resolve conflicts please :smile:

**Comment by tramuntanal** [comment]
I'm closing this PR in favor of #25  because it directly uses the omniauth gem. We'd rather use `decidim-idcat_mobile` gem.

**Pull request closed without merging**